### PR TITLE
Bypass decoy proteins when writing reversed seqs

### PR DESCRIPTION
**Pull request by jwertz01**
For proteins that are already decoys on input to reverse sequence script, don't output decoys of those. (Removal of XXX_XXX_ proteins from the output.) Issue was that these reversed reversed sequences had the same sequences as targets.
Tested and is working correctly.

### CODE REVIEW

File: src/main/java/edu/ucsd/workflow/ShuffleFASTA.java [hunk 1]
```diff
@@ -45,27 +45,42 @@ public static void main(String[] args) {
...
 					// insert the proper decoy prefix
 					line = line.substring(1).trim();
-					line = ">" + DECOY_PREFIX + "_" + line;
 					// if a previous sequence is still being processed,
 					// reverse it and write it to the output file
-					if (sequence != null && sequence.length() > 0)
+					if (
+						sequence != null && sequence.length() > 0 &&
+						(!prevProteinIsDecoy)
+
```
Comment (jjcarver) [inline src/main/java/edu/ucsd/workflow/ShuffleFASTA.java]: Why exactly is there a blank line here?

**Review by jjcarver — CHANGES_REQUESTED** [review]

File: src/main/java/edu/ucsd/workflow/ShuffleFASTA.java (hunk 1 above)
Comment (jwertz01) [reply src/main/java/edu/ucsd/workflow/ShuffleFASTA.java]: The line break is there to conform to the max line length for this script (which seems to be 80 characters).

Comment (jjcarver) [reply src/main/java/edu/ucsd/workflow/ShuffleFASTA.java]: I'm not referring to the line break between lines 70 and 71. That's fine. I'm referring to line 72, which is completely blank.

Comment (jwertz01) [reply src/main/java/edu/ucsd/workflow/ShuffleFASTA.java]: Removed the blank line

**Review by jjcarver — APPROVED** [review]

**Pull request merged**